spec/controllers/proposals_controller_spec.rb

```diff
@@ -229,9 +229,8 @@
       token = create(:api_token, step: approval)
       approval.user.add_delegate(create(:user))
 
-      post :approve, id: proposal.id, cch: token.access_token
+      get :approve, id: proposal.id, cch: token.access_token
```

Comment:
oh interesting! I think the source of my confusion is that we have one `post` spec at the top of the approval specs (which uses a token) and then a bunch of `get`s and then a bunch of `post`s. Do you think it makes sense to group them?